packages/manager/modules/pci/src/components/project/offer/notice/notice.template.html

```diff
@@ -0,0 +1,16 @@
+<small>
```

Comment:
I'm septical about this, I think the content of the legal text should be in a `small`, I don't think a link to the lega content should, but I'm not sure at all 😄

Reply:
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-small-element

Reply:
I'm not sure about it either 😕